quizz.md

```diff
@@ -0,0 +1,213 @@
+## Ruby Basics & Enumerables (meets Beauty and the Beast)
+
+
+### Question 1
+
+Define a method called `offerRose`, which should take one argument named person.
+
+When called, the method should `puts`, "Would you take this rose and help out
+an old beggar, X?", where X is the person passed into the method.
+
+Demonstrate calling the method with an argument of "young prince".
+
+Write your code here:
+```
+def offerRose (person)
+  puts "Would you take this rose and help out an old beggar, " + person + "?"
+end
+
+offerRose('young_prince')
+```
+### Question 2
+
+Assume the following hash:
+
+```
+town = {
+  residents: ["Maurice", "Belle", "Gaston"],
+  castle: {
+    num_rooms: 47,
+    residents: "Robby Benson",
+    guests: []
+  }
+}
+```
+
+Using Ruby, remove Belle from the town residents, and
+add her to the list of guests in the castle.
+
+Write your code here:
+```
+town[:castle][:guests] << town[:residents].delete("Belle")
+
+# ```
+# town[:residents].delete( "Belle" )
+# +town[:castle][:guests] = "Belle"
+# other possible solution? or is this is just deleting and creating a new value in the array?
```

Comment:
This other option would be ok, if:
1. you assigned what you deleted to a variable and used that during the addition to the castle
    - we want to "remember" the value via a variable, rather than by copying and pasting the string.
2. you added her to the castle, instead of overwriting the list of castle guests with Belle.
    - review `town` once your code has run.  What is the datatype for `town[:castle][:guests]` now?
e.g.
```
belle = town[:residents].delete( "Belle" )
town[:castle][:guests] << belle
```